tools/amp_ner/ner_to_csv.xml

```diff
@@ -0,0 +1,22 @@
+<tool id="ner_to_csv" name="NER to CSV" version="1.1.0">
+  <description>Convert NER output to CSV</description>
+  <requirements>
+  </requirements>
+  <command detect_errors="exit_code">
+    '$__tool_directory__/ner_to_csv_.py' '$amp_entities' '$amp_entities_csv';
+  </command>
+  <inputs>
+    <param name="amp_entities" type="data" format="ner" label="AMP Named Entities" help="AMP NER output file" ></param>
+  </inputs>
+  <outputs>
+    <data name="amp_entities_csv" format="csv" label="AMP Entities Extracted CSV"/>
+  </outputs>
```

Comment:
How about have the label as "AMP Entities CSV Converted"

Reply:
And done!